build.sbt

```
@@ -61,8 +53,40 @@ lazy val `akka-dependencies` =
         }
       },
       publishLocalConfiguration := publishLocalConfiguration.value.withOverwrite(true),
-      publishM2Configuration := publishM2Configuration.value.withOverwrite(true)
-    )
+      publishM2Configuration := publishM2Configuration.value.withOverwrite(true))
+    .aggregate(docs)
+
+lazy val docs = project
+  .enablePlugins(SitePreviewPlugin, AkkaParadoxPlugin, ParadoxSitePlugin, PublishRsyncPlugin)
+  .settings(
+    name := "Akka Dependencies",
+    previewPath := (Paradox / siteSubdirName).value,
+    Paradox / siteSubdirName := s"docs/akka-dependencies/${projectInfoVersion.value}",
+    projectInfoVersion := (if (isSnapshot.value) "snapshot" else version.value),
+    publish / skip := true,
```

This does the trick for build tool Scala versions:

```suggestion
    publish / skip := true,
    scalaVersion = "2.13.12",
```